server/report_builder/src/lib.rs

```diff
@@ -39,16 +39,16 @@ pub struct BuildArgs {
     #[clap(long)]
     pub query_default: Option<String>,
 
-    /// File name of the SQLite query.
-    /// If specified the query_postgres must be provided as well.
-    /// However, query_sqlite and query_postgres can point to the same file.
-    #[clap(long)]
-    pub query_sqlite: Option<String>,
-    /// File name of the Postgres query.
-    /// If specified the query_sqlite must be provided as well.
-    /// However, query_sqlite and query_postgres can point to the same file.
-    #[clap(long)]
-    pub query_postgres: Option<String>,
+    /// SQL query name.
+    /// This argument requires that there is either
+    /// - a single {query_sql}.sql file (for both Sqlite and Postgres)
```

Comment:
```suggestion
    /// - a single {query_name}.sql file (for both Sqlite and Postgres)
```
I think that's a bit more clearer naming (IMHO) would want to change the other comments to match of course.

Reply:
I used the name of the parameter, i.e. `query_sql` I put the sql in there to differenciate between the graphql queries (`query_gql` paramter). So think {query_sql} is more consitent.